www/script/test/test_realtimeRouting.js

```diff
@@ -0,0 +1,27 @@
+/*global define, describe, it, expect, beforeEach, afterEach, spyOn*/
+define(function (require) {
+    "use strict";
+
+    var $ = require("jquery"),
+        helpers = require('helpers'),
+        realtimerouting = require("realtimerouting"),
+        rtGlobal = require("rtGlobal");
+
+
+    describe("Add page init handler", function () {
+        it("page init handler has been executed", function (done) {
+            var page_id = "test_handler";
+            realtimerouting.addPageInitHandler(page_id, done);
+
+            helpers.getCurrentPage = function() { return page_id; }
+            realtimerouting.init();
```

Comment:
So the 'done' handler here should be invoked,  if it is not invoked - during realtimerouting.init() call - the test will fail(it's jasmine test callback)